source/brailleDisplayDrivers/papenmeier.py

```diff
@@ -152,7 +152,6 @@ def connectBluetooth(self):
 				key=lambda item: "bluetoothName" in item,
 			):
 				port = portInfo["port"]
-				hwID = portInfo["hardwareID"]  # noqa: F841
```

Comment:
@jcsteh or @LeonarddeR, I do not know braille driver code at all.
Would you be able to confirm that this variable was not meant to be used elsewhere but forgotten instead?

Reply:
Pretty sure this is safe to drop if not used elsewhere.